### ISSUE TITLE
[bug] Memory usage in dashboard seemingly incorrect

### ISSUE THREAD

**Issue by netthier** [siderolabs/omni-feedback#42]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

All my clusters in Omni are showing the same memory usage of exactly 876MB, e.g.:
![image](https://user-images.githubusercontent.com/66856670/236877156-35350ed8-d817-4ee3-b33e-e72258441ae3.png)

### Expected Behavior

Both clusters are running workloads and should have a memory usage higher than that.
There are pods running in both clusters that are using multiple GB of memory.
I would also expect the clusters to have different amount of memory usage due to running a different amount of software.

### Steps To Reproduce

1. Have a cluster in Omni
2. Start workloads in the cluster that use a lot of RAM
3. Wonder why the memory usage in Omni isn't budging

### What browsers are you seeing the problem on?

Firefox

### Anything else?

I've first reported this issue in Slack: https://taloscommunity.slack.com/archives/C04D4PDAJT0/p1683547740524229
I'm not sure how to get more detailed numbers, but k9s shows a memory usage percentage of 25%, which AFAIK comes from the metrics-server running in the cluster.
@smira mentioned this data being retrieved from `kubectl describe nodes`, so here is an excerpt from one of the nodes:
```
Name:               zir-01
Roles:              control-plane
Labels:             beta.kubernetes.io/arch=amd64
                    beta.kubernetes.io/os=linux
                    kubernetes.io/arch=amd64
                    kubernetes.io/hostname=zir-01
                    kubernetes.io/os=linux
                    node-role.kubernetes.io/control-plane=
Annotations:        node.alpha.kubernetes.io/ttl: 0
                    volumes.kubernetes.io/controller-managed-attach-detach: true
CreationTimestamp:  Wed, 05 Apr 2023 13:03:42 +0200
Taints:             <none>
Unschedulable:      false
Lease:
  HolderIdentity:  zir-01
  AcquireTime:     <unset>
  RenewTime:       Mon, 08 May 2023 18:20:43 +0200
Conditions:
  Type                 Status  LastHeartbeatTime                 LastTransitionTime                Reason                       Message
  ----                 ------  -----------------                 ------------------                ------                       -------
  NetworkUnavailable   False   Wed, 05 Apr 2023 13:04:46 +0200   Wed, 05 Apr 2023 13:04:46 +0200   CiliumIsUp                   Cilium is running on this node
  MemoryPressure       False   Mon, 08 May 2023 18:15:49 +0200   Sat, 06 May 2023 02:19:44 +0200   KubeletHasSufficientMemory   kubelet has sufficient memory available
  DiskPressure         False   Mon, 08 May 2023 18:15:49 +0200   Sat, 06 May 2023 02:19:44 +0200   KubeletHasNoDiskPressure     kubelet has no disk pressure
  PIDPressure          False   Mon, 08 May 2023 18:15:49 +0200   Sat, 06 May 2023 02:19:44 +0200   KubeletHasSufficientPID      kubelet has sufficient PID available
  Ready                True    Mon, 08 May 2023 18:15:49 +0200   Sat, 06 May 2023 02:19:44 +0200   KubeletReady                 kubelet is posting ready status
Addresses:
  InternalIP:  10.3.0.3
  Hostname:    zir-01
Capacity:
  cpu:                8
  ephemeral-storage:  129804Mi
  hugepages-2Mi:      0
  memory:             65825184Ki
  pods:               110
Allocatable:
  cpu:                7950m
  ephemeral-storage:  122229987535
  hugepages-2Mi:      0
  memory:             65526176Ki
  pods:               110
System Info:
  Machine ID:                 redacted
  System UUID:                redacted
  Boot ID:                    redacted
  Kernel Version:             6.1.25-talos
  OS Image:                   Talos (v1.4.1)
  Operating System:           linux
  Architecture:               amd64
  Container Runtime Version:  containerd://1.6.20
  Kubelet Version:            v1.27.1
  Kube-Proxy Version:         v1.27.1
PodCIDR:                      10.244.2.0/24
PodCIDRs:                     10.244.2.0/24
Non-terminated Pods:          (42 in total)
[...]
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource           Requests    Limits
  --------           --------    ------
  cpu                360m (4%)   0 (0%)
  memory             932Mi (1%)  0 (0%)
  ephemeral-storage  0 (0%)      0 (0%)
  hugepages-2Mi      0 (0%)      0 (0%)
Events:              <none>
```

**Comment by steverfrancis** [comment]
Confirmed this is a bug - the Omni display is showing the sum, for all nodes, of "Capacity.memory", less "Allocatable.memory".

So effectively this is just showing the memory reserved by K8s, and it never changes regardless of what pods are running.

It should be showing "Capacity.memory" less "Allocatable.memory" less "Allocated resources.memory".

Although this will even then only report memory allocation requests, not actual usage. I wonder if we should install/enable metrics server by default?

**Comment by netthier** [comment]
An alternative could be to just show the RAM usage as reported by Linux, i.e. as is already shown on the machine overview :thinking: 
I'd assume that the CPU and disk usage stats are currently being retrieved the same way.

**Comment by smira** [comment]
I think for this purpose, if we're showing what is available for scheduling, it should subtract CPU/memory requests. This will be more inline with pods limit.

**Comment by smira** [comment]
Fixed in 0.11
